server/matchmaker/search.py

```diff
@@ -40,8 +42,29 @@ def __init__(self, player, start_time=None, rating_prop='ladder_rating'):
             0: 0.8
         }
 
+    @property
+    def adjusted_rating(self):
+        """
+        Returns an adjusted mean with a simple linear interpolation between current mean and a specified base mean
+        """
+        if self.rating_prop=='ladder_rating':
```

Comment:
` == `

Reply:
What is the issue ? Can you please elaborate a little bit ?